.eslintrc

```diff
@@ -81,7 +81,7 @@
     "semi": [2, "always"],
     "space-before-blocks": [2, "always"],
     "space-before-function-paren": [2, "never"],
-    "space-infix-ops": 0,
+    "space-infix-ops": 2,
```

Comment:
Sorry, I also went with the executive decision to use `2` instead of `"error"` for consistency with other rules.

> The majority of web-ext code uses spaces around infix operators but this is not enforced. Let's enforce it by adding this rule to .eslintrc:

> ```yaml
"space-infix-ops": "error"
```
> &mdash; via https://github.com/mozilla/web-ext/issues/482#issue-176728517